src/orm/entities/user.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm'
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
```

Comment:
let's add createdAt and updatedAt colum. feel free to create a base entity with these columns and add id column to Base entity as well